nms/packages/magmalte/app/views/equipment/EnodebDetailSummaryStatus.js

```diff
@@ -122,6 +122,59 @@ export function EnodebStatus({refresh}: {refresh: boolean}) {
         value: enbInfo?.enb_state.ip_address ?? 'Not Available',
       },
     ],
+    [
+      {
+        category: 'SOFTWARE Version',
+        value: enbInfo?.enb_state.sw_version ?? 'Not Available',
+      },
+      {
+        category: 'Model Name',
+        value: enbInfo?.enb_state.model_name ?? 'Not Available', 
+      },
+      {
+        category: 'RF state',
```

Comment:
Please make a best effort to have capitalization be consistent.

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812870-cfd787f4-1982-4903-9b32-caea6ce5c59a.png)
modified,thank you!